### PR TITLE
[tests-only] use spaces dav path in `apiAuth` suite

### LINKED ISSUE #3022: use spaces webdav for `apiAuth` suite
### Description
- use spaces WebDAV URL for `apiAuth` suite from `oc/core`
- includes:
  - cors.feature (no need)
  - corsOc10Issue34679.feature (no need)
  - filesAppAuth.feature (no need)
  - tokenAuth.feature (no need)
  - webDavAuth.feature

### Related
- part of https://github.com/owncloud/ocis/issues/2854

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
use spaces DAV path in `apiAuth` suite

## Related Issue

- fixes https://github.com/owncloud/ocis/issues/3022

## How Has This Been Tested?

- :robot: 
- https://github.com/owncloud/ocis/pull/3025

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by amrita-shrestha** [comment]
LGTM

File: tests/acceptance/features/apiAuth/webDavAuth.feature [hunk 1]
```diff
@@ -5,14 +5,30 @@ Feature: auth
     Given user "Alice" has been created with default attributes and without skeleton files
 
   @smokeTest
-  Scenario: using WebDAV anonymously
+  Scenario Outline: using WebDAV anonymously
     When a user requests "/remote.php/webdav" with "PROPFIND" and no authentication
```
Comment (phil-davis) [inline tests/acceptance/features/apiAuth/webDavAuth.feature:9]:
```suggestion
    When a user requests "<dav_path>" with "PROPFIND" and no authentication
```

File: tests/acceptance/features/bootstrap/AuthContext.php [hunk 2]
```diff
@@ -133,6 +133,22 @@ public function userRequestsURLWith(string $url, string $method):void {
 		$this->sendRequest($url, $method);
 	}
 
+	/**
+	 * @When user :user requests :url with :method and no authentication
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/AuthContext.php:137]: I don't see where this new step is used.

**Review by SwikritiT — COMMENTED** [review]
LGTM other than already mentioned

File: tests/acceptance/features/apiAuth/webDavAuth.feature (hunk 1 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/apiAuth/webDavAuth.feature:9]: hmm...for this scenario, the outline is not useful. fixed it

File: tests/acceptance/features/bootstrap/AuthContext.php (hunk 2 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/bootstrap/AuthContext.php:137]: fixed

**Review by phil-davis — APPROVED** [review]

**Pull request merged**